docs/learn/glossary.md

```diff
@@ -0,0 +1,114 @@
+
+### DID wallets
+DID wallets are software libraries or end-user applications that wrap [DID providers](#did-providers) with additional capabilities. [3ID Connect](../authentication/wallets/3id-connect.md) is the most popular DID wallet SDK that allows users create, manage, and use a 3ID DID method with their existing blockchain wallets, and without needing to install any additional software.
+
+## Network
+
+### Clients
+Clients are software libraries that provide developer interfaces to a Ceramic [node](#nodes). Clients are responsible for [authenticating users](#authentication), providing [StreamType](#streamtypes)-specific interfaces for generating [genesis commits](#genesis-commit) and [signed commits](#signed-commit), and providing generic, streamtype-agnostic interfaces for loading or querying streams. A list of Ceramic clients can be found [here](./clients.md).
+
+### Nodes
+Nodes are software libraries that provide core protocol functionality for the Ceramic [network](#networks). Nodes are responsible for processing [stream](#streams) updates (in the form of [signed commits](#signed-commit) from [clients](#clients)), storing [state](#state) for the streams that it cares about, responding to queries, networking with other nodes, replicating streams across the network, and sending valid signed commits to an external [anchor service](#anchor-service) for generating [anchor commits](#anchor-commit).
+
+### Anchor service
+Anchor services are hosted "layer-2" services for Ceramic that generate [anchor commits](#anchor-commit) for many different [streams](#streams) in a scalable, low cost manner by batching many different stream transactions into a merkle tree, and including the merkle root into a transaction on a blockchain platform (currently [Ethereum](#ethereum)). This eliminates the need for each stream transaction to have its own corresponding blockchain transaction, which would be slower and more expensive.
+
+### Networks
+Networks are collections of Ceramic [nodes](#nodes) that share specific configurations and communicate over a dedicated [libp2p](#libp2p) topic. Networks are discrete from one another. [Streams](#streams) that exist on one network are not discoverable on or portable to another. Currently, Ceramic has three primary networks: [mainnet](#mainnet), [Clay Testnet](#clay-testnet), and [dev unstable](#dev-unstable).
+
+### Mainnet
+Mainnet is the Ceramic [network](#networks) used for production deployments. For more information on mainnet, see the Networks page.
+
+### Clay Testnet
+Clay Testnet is a Ceramic [network](#networks) used for community testing and development purposes. For more information on Clay Testnet, see the Networks page.
+
+### Dev Unstable
+Dev Unstable is a Ceramic [network](#networks) used by Ceramic core protocol developers for testing new protocol features and release candidates. It should be considered unstable and highly experimental.
```

Comment:
We also test release candidates on Clay. On dev unstable we test the most recent commits on the `develop` branch of js-ceramic.